src/main/java/com/greenfox/tribesoflagopus/backend/service/LoginService.java

```diff
@@ -38,27 +37,35 @@
       return ResponseEntity.badRequest().body(missingParameterStatus);
     }
 
-    if (!loginInput.getUsername().equals("Bond")){
+    if (loginInput == null) {
+      StatusResponse missingAllFields = StatusResponse.builder()
+              .status("error")
+              .message("Missing parameter(s): password, username!")
+              .build();
+      return ResponseEntity.badRequest().body(missingAllFields);
```

Comment:
You should generate your responses in your controller layer.
I create a card for refactoring this later.